src/main/java/org/jenkinsci/test/acceptance/po/Build.java

```diff
@@ -131,8 +131,15 @@ public boolean isInProgress() {
         }
```

Comment:
correct @jmdesprez and I discussed that outside of this PR.
The issue is that `inProgress` will return `false` if the build has not yet started `return state.equals(State.BUILDING) || state.equals(State.POST_PRODUCTION);`  but existing users of this may be expecting the old behaviour.

That said your comment prompted me to do a quick github search of this repo and CloudBees' proprietery ones and there are only 5 callers that I found - and all of those seem to be protected by `waitForStarted` (would need validation) with the exception of the single CLoudBees proprietry one, so I think the change makes sense. (it does after all match the name of the function!)

Reply:
Not sure why this was resolved—according to the above comment it is still a feasible suggestion? Something like

```java
    public boolean isInProgress() {
        JsonNode d = getJson();
        JsonNode inProgress = d.get("inProgress");
        if (inProgress != null) {
            // see https://github.com/jenkinsci/jenkins/pull/6829
            return inProgress.booleanValue();
        }

        // TODO delete when inProgress becomes part of the LTS
        if (result != null) {
            return false;
        }
        if (!hasStarted()) {
            return false;
        }
        // Please note that the test below denote a completed build, not a finished build
        return d.get("building").booleanValue() || d.get("result") == null;
    }
```
which would clarify that going forward we expect this page object method to have identical behavior to the Java method inside Jenkins, and that the complicated logic is just a workaround for older versions.

Reply:
test